### PR TITLE
feat(Config): add rms assignment package resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add rms assignment package resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
   add rms assignment package resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/rms/' TESTARGS='-run TestAccAssignmentPackage_basic' 
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/rms/ -v -run TestAccAssignmentPackage_basic -timeout 360m -parallel 4 
=== RUN   TestAccAssignmentPackage_basic 
=== PAUSE TestAccAssignmentPackage_basic 
=== CONT  TestAccAssignmentPackage_basic 
--- PASS: TestAccAssignmentPackage_basic (76.77s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rms       76.815s
```

### CODE REVIEW

File: huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+func buildCreateAssignmentPackageRequestBodyParameter(rawParams interface{}) []map[string]interface{} {
+	if rawArray, ok := rawParams.([]interface{}); ok {
+		if len(rawArray) == 0 {
+			return nil
+		}
+
+		rst := make([]map[string]interface{}, len(rawArray))
+		for i, v := range rawArray {
+			raw := v.(map[string]interface{})
```
Comment (hmbig2) [inline huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go]:
```suggestion
			if raw,ok := v.(map[string]interface{});ok{
```

File: huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+		Target:       []string{"CREATE_SUCCESSFUL", "ROLLBACK_SUCCESSFUL"},
+		Pending:      []string{"CREATE_IN_PROGRESS"},
+		Refresh:      rmsAssignmentPackageStateRefreshFunc(createAssignmentPackageClient, cfg, d.Id()),
+		Timeout:      d.Timeout(schema.TimeoutCreate),
+		Delay:        10 * time.Second,
+		PollInterval: 10 * time.Second,
+	}
+
+	if _, err = stateConf.WaitForStateContext(ctx); err != nil {
+		return diag.Errorf("error waiting for RMS Assignment Package (%s) to be created: %s", id.(string), err)
```
Comment (hmbig2) [inline huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go]:
```suggestion
		return diag.Errorf("error waiting for RMS assignment package (%s) to be created: %s", id.(string), err)
```

Comment (houpeng80) [reply huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go]: done

File: huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/rms/resource_huaweicloud_rms_assignment_package.go]: done

**Pull request merged**